components/onboarding-modal/OnboardingStepsProgress.js

```diff
@@ -0,0 +1,78 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Flex } from '@rebass/grid';
+
+import { Span } from '../Text';
+import StepsProgress from '../StepsProgress';
+import { Router } from '../../server/pages';
+
+const StepLabel = styled(Span)`
+  text-transform: uppercase;
+  text-align: center;
+`;
+
+StepLabel.defaultProps = {
+  color: 'black.400',
+  fontSize: 'Tiny',
+  mt: 1,
+};
+
+const steps = [{ name: 'Welcome' }, { name: 'Administrators' }, { name: 'Contact' }];
+
+const params = {
+  0: {
+    routerStep: undefined,
+  },
+  1: {
+    routerStep: 'administrators',
+  },
+  2: {
+    routerStep: 'contact',
+  },
+};
+
+class OnboardingStepsProgress extends React.Component {
+  static propTypes = {
+    step: PropTypes.number,
+    slug: PropTypes.string,
+  };
+
+  constructor(props) {
+    super(props);
+  }
```

Comment:
**[:ice_cream: Details & personal preferences]** Adding that on all components is not required

```suggestion
```

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>